lib/contentful/client.rb

```diff
@@ -43,6 +45,24 @@ def self.get_http(url, query, headers = {}, proxy = {})
       end
     end
 
+    # @param [Hash] given_configuration
```

Comment:
Might make sense to actually take stuff from <https://github.com/contentful/contentful.rb#client-configuration-options> and put it here.